src/plugins/node/prof/compat-inl.h

```diff
@@ -0,0 +1,381 @@
+/*******************************************************************************
+ * Copyright 2014, StrongLoop Inc.
```

Comment:
Agreed. All of the StrongLoop copyright headers should be changed to IBM and updated to use the same copyright header formatting as the other source code in this repo.